integration/dataservice-hosting-tests/tests-integration/tests/src/test/java/org/wso2/ei/dataservice/integration/test/odata/ODataSuperTenantUserTestCase.java

```diff
@@ -194,7 +199,9 @@ private static int sendPUT(String endpoint, String content, String acceptType) t
             httpPut.setEntity(httpEntity);
         }
         HttpResponse httpResponse = httpClient.execute(httpPut);
-        return httpResponse.getStatusLine().getStatusCode();
+        int statusCode = httpResponse.getStatusLine().getStatusCode();
+        httpPut.releaseConnection();
```

Comment:
Let's have a try and finally block and, release the connection inside the finally block. So that we can make sure the connection will be released whether an exception is thrown or not.

Shall we can close the httpclient as well inside the finally block.
`httpClient.close();`